main.py

```diff
@@ -1,372 +1,37 @@
-        return
-    resp = requests.get("https://radio17.pl/api/schedule")
-    resp_json = resp.json()
-    pages = []
-    if day.lower() == "short":
-        short_embed = discord.Embed(title="Dziś jeszcze gramy:")
-        for broadcast in resp_json[week_days[datetime.datetime.today().weekday()]]:
-            if int(broadcast["end_hour"][:2]) > datetime.datetime.today().hour:
-                short_embed.add_field(name=broadcast["title"],
-                                      value=broadcast["start_hour"] + " - " + broadcast["end_hour"], inline=False)
-        await ctx.send(embed=short_embed)
-        return
-    if day.lower() not in week_days:
-        await ctx.send("Wpisz poprawny dzień tygodnia w języku angielskim!")
-        return
-    for broadcast in resp_json[day]:
-        embed = discord.Embed(title=broadcast["title"])
-        embed.add_field(name="Nazwa audycji:", value=broadcast["title"], inline=False)
-        embed.add_field(name="Od - do:", value=broadcast["start_hour"] + " - " + broadcast["end_hour"], inline=False)
-        if broadcast["description"] is not None:
-            embed.add_field(name="Opis:", value=broadcast["description"], inline=False)
-        embed.set_image(url=broadcast["player_image"])
-        pages.append(embed)
-    message = await ctx.send(embed=pages[0])
-    await message.add_reaction('⏮')
-    await message.add_reaction('◀')
-    await message.add_reaction('▶')
-    await message.add_reaction('⏭')
-
-    def check(reaction, user):
-        return user == ctx.author
-
-    i = 0
-    reaction = None
-    while True:
-        if str(reaction) == '⏮':
-            i = 0
-            await message.edit(embed=pages[i])
-        elif str(reaction) == '◀':
-            if i > 0:
-                i -= 1
-                await message.edit(embed=pages[i])
-        elif str(reaction) == '▶':
-            if i < len(pages) - 1:
-                i += 1
-                await message.edit(embed=pages[i])
-        elif str(reaction) == '⏭':
-            i = len(pages) - 1
-            await message.edit(embed=pages[i])
-        try:
-            reaction, user = await bot.wait_for('reaction_add', timeout=30.0, check=check)
-            await message.remove_reaction(reaction, user)
-        except:
-            break
-    await message.clear_reactions()
-
-
 @bot.event
-async def on_command_error(ctx, error):
+async def on_command_error(ctx, error): # w przypadku gdy komenda jest na cooldownie wysyla wiadomosc ile sekund pozosalo do jej nastepego uzycia
     if isinstance(error, commands.errors.CommandOnCooldown):
-        await ctx.channel.send('This command is on a %.2fs cooldown' %
-                               error.retry_after)
+        await ctx.channel.send(f'This command is on a {error.retry_after:.2f}s cooldown')
+
 
 
 @bot.event
 async def on_message(message):
-    if message.author == bot.user:
+    if message.author == bot.user:  # ignoruje wiadomosci bota, by bot na swoje same nie odpowiadal
```

Comment:
```suggestion
    # ignoruje wiadomości bota, by bot na swoje same nie odpowiadał
    if message.author == bot.user: 
```
Jeżeli już dajesz komentarze to najlepiej nad linią do której jest on przypisany, wszędzie się tak raczej przyjęło. Dlaczego? Jak robisz później zmianę to w historii gita widać że zmieniła się cała linia - jak robisz zmianę tylko komentarza to widać że zmieniłęś komentarz. Stricte pod taki komfort pracy